resources/lang/en/beatmapsets.php

```diff
@@ -52,6 +52,11 @@
             'favourite' => 'Favourite this beatmapset',
             'unfavourite' => 'Unfavourite this beatmapset',
             'favourited_count' => '+ 1 other!|+ :count others!',
+
+            'login_required' => [
+                'top' => 'Sign In',
+                'bottom' => 'for more features on this page',
```

Comment:
Maybe `to access more features`? Alternatively `for more features`? but that feels maybe too short..